Dockerfile-3.5

```diff
@@ -8,7 +8,7 @@ RUN pip install -r requirements.txt -r requirements-test.txt
 
 COPY . /opt/project
 
-RUN pip install -e .
```

Comment:
These dockerfiles are used for in-place debug, and have volumes configured in docker-compose files, so removing -e from them will break running from docker-compose.

The same is for docker-entrypoint file.

So I don't have any objections to Makefile change and Appveyor config, but I don't think changes to docker related things are really required